Update intro paragraph (#1139)
filecoin-project/filecoin-docs: docs/README.md

# Filecoin Documentation

> **Filecoin is a peer-to-peer network that allows anyone to store and retrieve files on the internet. Built-in economic incentives ensure that files are stored and retrieved reliably and continuously for however long a user specifies.**

Reviewer: ```suggestion
Filecoin is a peer-to-peer network that allows anyone to store and retrieve data on the internet. Built-in economic incentives ensure that files are stored and retrieved reliably and continuously for however long a user specifies.

```

> Available storage capacity and the pricing for storing and retrieving files are not controlled by any single company. Instead, the Filecoin storage network facilitates open markets for storing and retrieving files that anyone can participate in. It uses its native cryptocurrency (FIL) to faciliate these transactions.

Reviewer: ```suggestion
No single company or organization controls available storage capacity and the pricing for storing and retrieving files. Instead, the Filecoin storage network facilitates open markets for storing and retrieving files that anyone with access to the internet can participate in. It uses its native cryptocurrency (FIL) to facilitate these transactions.
```
